client/new-stack-editor/lib/views/wizard/providerselectionview.coffee

```diff
@@ -0,0 +1,67 @@
+kd = require 'kd'
+JView = require 'app/jview'
+globals = require 'globals'
+Tracker = require 'app/util/tracker'
+
+
+module.exports = class ProviderSelectionView extends JView
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass ?= 'provider-selection'
+
+    super options, data
+
+    @createProviders()
+
+
+  createProviders: ->
+
+    { providers } = globals.config
+
+    supportedProviders = (Object.keys providers).filter (p) ->
+      return p  if providers[p].supported
```

Comment:
I agree but I usually prefer to use the way everybody can understand without a knowledge. TIL `pickBy` we can use it further.